<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Distance</class>
 <widget class="QWidget" name="Distance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>844</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>389</width>
    <height>844</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>844</width>
    <height>844</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>视力测试</string>
  </property>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>400</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>近视力测试</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>400</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>40</horstretch>
     <verstretch>60</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>远视力测试</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>801</width>
     <height>611</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>欢迎使用本视力检测工具！在进行测试前，请注意以下事项：
请确定要测试的眼睛，并用同侧手掌捂住未被测的同侧眼，以确保测试结果准确。
请在正常光线下使用本工具，以避免测试结果受到影响。
当测试时，请勿出现多人手掌在摄像头前，以确保测试结果准确。

测试步骤如下：
点击“视力检测”按钮，开始测试。
当出现C视标后，请用闲置手的食指与中指一起指出自己看到的缺口方向。
如果看不清，则两张手掌都张开。
重复上述操作，直到界面显示“测试完毕”。
查看自己的测试结果。

注意事项：
如果系统连续多次判断错误，请重新进行测试。

希望本使用说明能够帮助您进行准确的视力检测，如有任何问题，请联系我们的客服。</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
